### PR TITLE
Fixes #11033 - adding missing params to auth_source ldap API docs

### PR DESCRIPTION
**Pull request by mbacovsky**

### CODE REVIEW

File: app/controllers/api/v2/auth_source_ldaps_controller.rb [hunk 1]
```diff
@@ -32,6 +32,9 @@ def show
           param :onthefly_register, :bool
           param :usergroup_sync, :bool, :desc => N_("sync external user groups on login")
           param :tls, :bool
+          param :groups_base, String, :desc => N_("group's base DN")
+          param :server_type, [AuthSourceLdap::SERVER_TYPES.keys], :desc => N_("type of the LDAP server")
```
Comment (domcleal) [inline app/controllers/api/v2/auth_source_ldaps_controller.rb]: Why the extra array?  .keys should be an array already.

File: app/controllers/api/v2/auth_source_ldaps_controller.rb [hunk 2]
```diff
@@ -32,6 +32,9 @@ def show
           param :onthefly_register, :bool
           param :usergroup_sync, :bool, :desc => N_("sync external user groups on login")
           param :tls, :bool
+          param :groups_base, String, :desc => N_("group's base DN")
```
Comment (domcleal) [inline app/controllers/api/v2/auth_source_ldaps_controller.rb]: Nitpick, remove the apostrophe to match the UI.

File: app/controllers/api/v2/auth_source_ldaps_controller.rb [hunk 3]
```diff
@@ -32,6 +32,9 @@ def show
...
+          param :server_type, [AuthSourceLdap::SERVER_TYPES.keys], :desc => N_("type of the LDAP server")
+          param :ldap_filter, String
```
Comment (domcleal) [inline app/controllers/api/v2/auth_source_ldaps_controller.rb]: Maybe add a desc?

**Comment by mbacovsky** [comment]
Updated, @domcleal

**Comment by domcleal** [comment]
[test]

**Comment by domcleal** [comment]
Merged as 66f3267 for 1.9, thanks @mbacovsky!

**Pull request closed without merging**